crates/js_api/src/gui/mod.rs

```diff
@@ -635,23 +628,19 @@ _ _: 0 0;
     }
 
     pub async fn initialize_gui(deployment_name: Option<String>) -> DotrainOrderGui {
```

Comment:
do we still need this initialize functions?

Reply:
these are used for tests to convert that initializer call into a one liner. i think we can keep this